src/pages/Send/index.js

```diff
@@ -100,6 +100,17 @@ class Send extends Component {
     };
   }
 
+  flipInputOutput = () => {
+    const { state } = this;
+    this.setState({
+      inputValue: state.outputValue,
+      outputValue: state.inputValue,
+      inputCurrency: state.outputCurrency,
+      outputCurrency: state.inputCurrency,
+      lastEditedField: state.lastEditedField === INPUT ? OUTPUT : INPUT
```

Comment:
- `lastEditedFied` should stay the same
- `inputCurrency` and `outputCurrency` should switch
- if `lastEditedField` is `inputValue`, `inputValue` should be `outputValue`, and `outputValue` should be `undefined` or `0`. 
- ^ vice versa

Then recalcForm will take care of the rest.